src/config/config-filter.c

```diff
@@ -37,8 +37,23 @@ static bool config_filter_match_rest(const struct config_filter *mask,
 	if (mask->local_name != NULL) {
 		if (filter->local_name == NULL)
 			return FALSE;
-		if (dns_match_wildcard(filter->local_name, mask->local_name) != 0)
+		/* Handle multiple names seperated by spaces in local_name
+		 * Ex: local_name "mail.domain.tld domain.tld mx.domain.tld" { ... } */
+		if (strstr(mask->local_name, " ") != NULL) {
```

Comment:
this if is useless, t_strsplit_spaces will return valid array in any case, so you could just leave the code below and remove the if completely.